### ISSUE TITLE
Suggestion to rename a metric

### ISSUE THREAD

**Issue by germonprez** [chaoss/wg-risk#167]
[Defect Resolution Time](https://chaoss.community/metric-defect-resolution-time/)	to	Defect Resolution Duration

This comes from the Common WG who has been looking at how we represent 'time' in our metrics in a consistent way.

**Comment by sophia-IV** [comment]
Im Ok with this change and generally supportive of consistent language

**Comment by sgoggins** [comment]
@germonprez : what is the process we are following to do this? I am unsure with website revisions if we should make the change or wait. @klumb ?

**Comment by germonprez** [comment]
I believe that if it is not changing the nature of the metric, you can simply make the change without requiring any sort of review. The change should be reflected on the website.

**Comment by klumb** [comment]
Agree, name changes that don't change the nature of the metric don't need a review. You may still want to document it in the release notes though. The name change will require a change on the website. Just let me know when the PR is accepted. Make sure that the name change is reflected in the header, file name, and all other places in the repo where the metric is referenced.
